app/models/organization.rb

```diff
@@ -0,0 +1,13 @@
+class Organization < ApplicationRecord
+  has_many :users
+  has_one :address, as: :addressable
+
+  validates :name, presence: true
```

Comment:
Since the null: false is set for :org_type in Organizations should the validation be set to presence: true

 ```suggestion
  validates :name, :org_type, presence: true
```

Reply:
Ok, sounds right.